src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -831,12 +831,8 @@
     <Content Include="azurefunctions\functions.js">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="azurefunctions\http\request.js">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-    <Content Include="azurefunctions\http\response.js">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
+    <Content Include="azurefunctions\http\request.js" />
+    <Content Include="azurefunctions\http\response.js" />
```

Comment:
undo this change. It's annoying, but these files need to be marked as always copy. Every time VS updated the references it does this for some reason that I don't know.